docs/source/common/lightning_cli.rst

```diff
@@ -683,6 +716,12 @@ And the same through command line:
 
     $ python train.py --optimizer.class_path=torch.optim.Adam --optimizer.init_args.lr=0.01
 
+Optionally, the command line can be simplified for PyTorch built-in `optimizers` and `schedulers`:
```

Comment:
It sounds to me like this should start the "Optimizers and learning rate schedulers" section. Also, let's add subtitles here to make it easier to find

Built in schedulers & optimizers
-------------------------------
If You need pytorch built in optimizers or scehdulers -> do this.

Register your own bla bla
-------------------------

Configure optimizers from CLI
------------------------------